src/component/Account/Account.js

```diff
@@ -6,34 +6,42 @@ import { ContractContext } from "../../hooks/contract";
 
 import Spinner from "../utils/Spinner";
 import AccountCard from "./AccountCard/AccountCard";
+import Generation from "../Generation/Generation"
 
 export default () => {
   const nearContext = useContext(NearContext);
   const useContract = useContext(ContractContext);
-  const { corgis, loading } = useContract;
-  useEffect(() => window.localStorage.removeItem("create"));
+  const { loading, getCorgisList } = useContract;
+  let corgiUpdateKey = 0;
```

Comment:
what is this for? I could not understand clearly.

Reply:
This was so that when the corgi would be created, the key would change, which was then supposed to trigger useEffect.
I changed it so that UseEffect is looking for changes in the ContractContext